src/PhpSlang/Collection/Generic/Component/PartitionableCollection.php

```diff
@@ -110,6 +111,8 @@
      * @param Collection $pairs
      *
      * @return Closure
+     *
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
```

Comment:
it's a deliberate decision - in this case we use hashmaps map function which can take a function with arguments - 1st stands for value, 2nd for a key (which is optional and usually not used). In this specific case we want ONLY key, and don't bother about content